app/connector/sql/src/main/java/io/syndesis/connector/sql/customizer/SqlConnectorCustomizer.java

```diff
@@ -104,12 +105,20 @@ private void doBeforeProducer(Exchange exchange) {
 
     private void doAfterProducer(Exchange exchange) {
 
+        System.out.println("AFTER PRODUCER " + exchange.getException());
```

Comment:
Can we use the (debug) log here instead, would give us the full stack trace?

Reply:
y these are just debug statements that need to be removed before removing the WIP label. Thx for catching it though.